examples/miscellaneous/miscellaneous_ex2/miscellaneous_ex2.C

```diff
@@ -543,9 +548,9 @@ void add_M_C_K_helmholtz(EquationSystems & es,
 
   // Compute the scale values for the different operators.
   const Number scale_stiffness (1., 0.);
-  const Number scale_damping (0., omega);
-  const Number scale_mass (-k*k, 0.);
-  const Number scale_rhs (0., -(rho*omega));
+  const Number scale_damping=I*omega; // I is imaginary unit (from complex.h)
```

Comment:
Not portably, I'm afraid.  I is the imaginary unit from complex.h in C99, which isn't guaranteed to be supported by or compatible with C++ compilers.  In fact, technically if you don't explicitly ```#include "complex.h"```, you just expect it to be included under the hood by ```#include <complex>```, then "I" is just polluting the global namespace in a C++ context, so any compiler/stdlib that *does* support it is violating the standard!

This appears to be why the Civet --enable-complex build barfed.

You can write 1i (after using the complex_literals sub-namespace) in C++14, but we don't demand users have C++14 support yet, and if you want to use i in C++11 then I'm afraid you'll have to define it yourself.

Reply:
ok, I will use an own variable. Thanks for the explanation.